src/main/java/seedu/address/logic/commands/AddNotesCommand.java

```diff
@@ -22,28 +28,55 @@ public class AddNotesCommand extends Command {
             + PREFIX_NOTES + "[NOTES]\n"
```

Comment:
Minor nit, I think can consider changing the command word from addNotes to snakecase form, to match the same style as our future make_appt commands

Reply:
for eg changing to add_notes